install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3141,6 +3141,9 @@ authentication:
         enabled: true
         # -- SPIRE namespace to install into
         namespace: cilium-spire
+        # -- SPIRE configuration image
+        spireConfig:
```

Comment:
nit -- this should be called `init`, since it matches the container name.

@meyskens can we just use spire-agent? does it have a shell?